packages/mgt-components/src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -319,6 +319,15 @@ export class MgtTasks extends MgtTemplatedComponent {
    */
   public taskFilter: TaskFilter;
 
+  /**
+   * Get the scopes required for tasks
+   *
+   * @static
+   * @return {*}  {string[]}
+   * @memberof MgtTasks
+   */
+  public static requiredScopes: string[] = ['group.read.all', 'group.readwrite.all', 'tasks.read', 'tasks.readwrite'];
```

Comment:
tasks uses `mgt-people` and `mgt-people-picker`

Reply:
Updated